src/plugins/discover_shared/public/services/discover_features/types.ts

```diff
@@ -30,8 +32,26 @@ export interface ObservabilityLogsAIAssistantFeature {
   render: (deps: ObservabilityLogsAIAssistantFeatureRenderDeps) => JSX.Element;
 }
 
+/** ***************************** Security **********/
```

Comment:
nit: Security Solution (not to be confused with our security plugin)....don't think anyone would...so super nit 😁

Reply:
🤣 .. Done here:  7af29690d18